README.md

```diff
@@ -1,5 +1,5 @@
 [![][ButlerImage]][website] 
-
+Refactored by Omar AbdulAziz -- SENG371
```

Comment:
Please revert changes to this file

Reply:
![image](https://user-images.githubusercontent.com/22069922/79448392-a58be780-7fe1-11ea-8148-ac2455bc0889.png)
Was this the expected result of this change? really?  :thinking: